core/src/main/java/org/apache/accumulo/core/clientImpl/NamespaceOperationsImpl.java

```diff
@@ -163,7 +166,11 @@ public void delete(String namespace) throws AccumuloException, AccumuloSecurityE
   public void rename(String oldNamespaceName, String newNamespaceName)
       throws AccumuloSecurityException, NamespaceNotFoundException, AccumuloException,
       NamespaceExistsException {
-
+    checkArgument(oldNamespaceName.length() <= MAX_NAMESPACE_LEN,
```

Comment:
Why does the old namespace need to be less than the len?  If the old namespace was created in an older version of Accumulo then it could be over the limit.

Reply:
@keith-turner Thanks for looking things over. I might be misunderstanding ctubbsii's feedback on this one. I thought the intention in the new version was to essentially force user's to rename namespaces to something that conforms to the 1024 char limit.

Reply:
@cradal No. Sorry for the confusion. The intention was:

1. prevent the creation of new objects over the limit, and
2. log a warning when we discover any object over the limit (with the understanding that this would only be possible for objects created before the limit was imposed)